util/fibers/submit_entry.h

```diff
@@ -194,14 +194,22 @@ class SubmitEntry {
     sqe_->cancel_flags = flags;
   }
 
-  io_uring_sqe* sqe() {
-    return sqe_;
+  void PrepMadvise(void* addr, off_t len, int advice) {
```

Comment:
Where is this called?

Reply:
It's not called in helio. It's part of helio's iouring API

Reply:
gotcha